addons/sys_godmode/fnc_getUnitsBIChannel.sqf

```diff
@@ -15,30 +15,27 @@
  * Public: No
  */
 
-private _units = [];
+private _units = allPlayers select {!(_x isKindOf "HeadlessClient_F")};
 
 switch (currentChannel) do {
-    case 0: { _units = allPlayers; };    // Global
-    case 1: {                            // Side
-        private _side = side acre_player;
-        _units = allPlayers select {(_side == (side _x)) && {(alive _x) || {[_x] call acre_api_fnc_isSpectator}}};
+    case 1: { // Side
+        private _side = side player;
+        _units = _units select {(_side == (side _x)) || {_x in ACRE_SPECTATORS_LIST}};
     };
-    case 3: {                            // Group
-        _units = (units group acre_player) select {(alive _x) || {[_x] call acre_api_fnc_isSpectator}};
+    case 3: { // Group
+        _units = units (group player);
```

Comment:
Are spectator units in a different group? What about players triggering spectator mode through the debug console?

Reply:
> Are spectator units in a different group?

That depends on the spectator system. There is no change here regarding spectators really, you just had to add it because you were filtering `alive`. But we now want dead units to hear as well, after some discussion it makes sense, since it is "admin talk".

> What about players triggering spectator mode through the debug console?

They'll just hear group. Again, no change from before. I don't think we can make this flexible enough, spectator systems can handle it if wanted.